templates/consul_asg.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Why is this here? Doesn't look like this has any place being in the generic terraform microservice module(!)